backend/src/appointment/controller/calendar.py

```diff
@@ -397,11 +397,17 @@ def list_events(self, start, end):
             if status == 'cancelled' or transparency == 'transparent':
                 continue
 
+            vevent = e.vobject_instance.vevent
+
+            # Ignore events with missing datetime data
+            if not vevent.dtstart or (not vevent.dtend and not vevent.duration):
```

Comment:
Can we add if not vevent too just in case 😅

Reply:
Of course! Good thinking 👍🏻